jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java

```diff
@@ -19,10 +44,18 @@ public BaseAction(Tween tween) {
         subActions.toArray(actions);
     }
 
+    /**
+     * Used to add the tween actions only if they are types of anim #{@link Action} class.
+     * Otherwise if they are in a raw tween state #{@link Tween}.
+     * then fetch the children actions of this tween & add them to subset of actions collection.
```

Comment:
In javadoc , the "&" character introduces an HTML entity. To avoid a javadoc error, use "&amp;" or (even better) spell it out as "and".